package.json

```diff
@@ -252,6 +252,10 @@
                     "group": "0_navigation@4",
                     "_comment": "see https://github.com/microsoft/vscode-references-view/blob/f63eaed9934ca5ecc8f3fb3ca096f38c6e5e181f/package.json#L162"
                 },
+                {
+                    "command": "clangd.switchheadersource",
+                    "group": "0_navigation@5"
+                },
```

Comment:
We should add `"when": "resourceLangId == cpp"`, to avoid this context menu item showing up in non-C++ files.

(Bonus: `"clang.ast"` should really have this condition to. If you'd like, feel free to submit a separate PR adding that.)

Reply:
Thank you @HighCommander4. I've added condition now to limit this to C++ files.